src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/DirectoryDAO.scala

```diff
@@ -14,9 +14,9 @@ import org.broadinstitute.dsde.workbench.sam.util.SamRequestContext
 trait DirectoryDAO extends RegistrationDAO {
   def createGroup(group: BasicWorkbenchGroup, accessInstructionsOpt: Option[String] = None, samRequestContext: SamRequestContext): IO[BasicWorkbenchGroup]
   def loadGroup(groupName: WorkbenchGroupName, samRequestContext: SamRequestContext): IO[Option[BasicWorkbenchGroup]]
-  def loadGroups(groupNames: Set[WorkbenchGroupName], samRequestContext: SamRequestContext): IO[Stream[BasicWorkbenchGroup]]
```

Comment:
Stream is renamed to `LazyList`

Reply:
that's ridiculous, stream is so much better